### ISSUE TITLE
deprecation warnings: Invalid escape sequence

### ISSUE THREAD

**Issue by mircealungu** [kurtmckee/feedparser#158]
I'm getting a lot of these: 
```

...env/lib/python3.7/site-packages/feedparser-5.2.1-py3.7.egg/feedparser.py:217
...env/lib/python3.7/site-packages/feedparser-5.2.1-py3.7.egg/feedparser.py:217: DeprecationWarning: invalid escape sequence \d

...env/lib/python3.7/site-packages/feedparser-5.2.1-py3.7.egg/feedparser.py:236
...env/lib/python3.7/site-packages/feedparser-5.2.1-py3.7.egg/feedparser.py:236: DeprecationWarning: invalid escape sequence \s

```

Anybody else?

**Comment by jgarbers** [comment]
Yes, while running tests, I get a couple dozen, and all those the warnings make the "real" results scroll away.

**Comment by kurtmckee** [comment]
I'll get this taken care of. Thanks for bringing this to my attention!

**Comment by mircealungu** [comment]
Hi @kurtmckee . Did you get the chance to look into this?

**Comment by Evolter** [comment]
Hi, it was bothering me as well, a quick search and I ended up here.
It seems that it has already been fixed in #123 we just need a new release (#191).

**Comment by kurtmckee** [comment]
6.0.0b2 has been released. I think this problem has been resolved. Please re-open if there are still escape sequence warnings.
